frontend/public/i18next-parser.config.js

```diff
@@ -1,14 +1,22 @@
+const PARSER_SUPPORTED_LOCALES = {
+  en: 'English',
+};
+
+/* eslint-disable no-unused-vars, @typescript-eslint/no-unused-vars */
```

Comment:
Oh, we shouldn't be referencing variables in the parser config in the app. I think we need to just make the locales map a separate variable in the language selector modal.

Reply:
@spadgett - We're also using it in webpack.config.ts - Joachim had it set up to process Chinese a certain way when it's available. Are you ok with maintaining it in two places?